spec/unit/filter_spec.rb

```diff
@@ -116,18 +116,18 @@
   describe "#filter_for" do
     it "returns the correct filter" do
       filter = Filter.new([
-        "/unmanaged_files/files/name=/home/alfred",
-        "/unmanaged_files/files/name=/var/cache",
-        "/changed_managed_files/files/changes=md5,size"
+        "/unmanaged_files/name=/home/alfred",
```

Comment:
Use 2 spaces for indentation in an array, relative to the first position after the preceding left parenthesis.